servermon/puppet/templates/query.html

```diff
@@ -42,6 +42,7 @@
   a.selector-chooseall:hover, a.selector-clearall:hover { color:#036; }
   a.selector-chooseall { width:7em; background:url({% admin_media_prefix %}img/admin/selector-addall.gif) left center no-repeat; }
   a.selector-clearall { background:url({% admin_media_prefix %}img/admin/selector-removeall.gif) left center no-repeat; }
+  select#id_facts, select#id_hosts { height: auto; min-height: 250px; width:75%; resize: vertical; }
```

Comment:
These seem like ids added by admin javascript in the DOM and not via HTML/JS controlled by the project. They also seem to longer exist in master (at least for me in chrome I can not find such elements). Which makes me think this is a fragile approach and it is better avoided